examples/with-changesets/packages/eslint-config/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@acme/eslint-config",
+  "version": "0.0.0",
+  "private": true,
+  "files": [
+    "library.js",
+    "next.js",
+    "react-internal.js"
+  ],
```

Comment:
I'm tempted to remove the files key just to prevent confusion. This is only used if you're packing the package to determine which files get included in the published package. It doesn't actually have anything to do with making the files available to be imported from in the way we're using it here - which I'll bet is what some may assume by just looking at this. 

But not blocking, we can hit that in a follow up!